### PR TITLE
add sponsorship package

### PR DESCRIPTION
**Pull request by keywordnew**
Some changes are needed before merge:
* We need a designated contact.

### CODE REVIEW

File: docs/sponsorship-package.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Types of sponsorship
+You or your organization can be either a _cash_ or a _venue_ sponsor.
+
+You can go either month-to-month or be a **12-month Exclusive Sponsor**.
+
+The **12-month Exclusive Sponsor** option is only available if all other sponsorships for your chosen type of sponsorshop (that is, either _cash_ or _venue_) at the time are "Month-to-month". In the event that an Exclusive Sponsor is secured, NodeSchoolYVR will not entertain any other sponsorship agreements of that type for the duration of the 12-month term.
+
+## Cash sponsors
+NodeSchoolYVR offers one benefits tier to any and all cash sponsors. Sponsorship funds with be used for, but is not limited to, food, drinks, and operational costs.
```
Comment (codeekage) [inline docs/sponsorship-package.md]: typo with -> will

**Review by codeekage — CHANGES_REQUESTED** [review]

**Comment by Qard** [comment]
Looks pretty good. One thing to consider is how Gastown Gang stuff will impact this in the future. We should, at the least, communicate the existence of Gastown Gang when speaking to potential sponsors and have some plan on how to migrate their sponsorship in the future.

File: docs/sponsorship-package.md (hunk 1 above)
Comment (keywordnew) [reply docs/sponsorship-package.md]: Thanks for reviewing!

**Comment by keywordnew** [comment]
We can def communicate the existence of Gastown Gang when speaking to sponsors.

I can push up some edits to include a mention of Gastown Gang. But without public information about Gastown Gang (and its responsibilities around sponsorship), I'm concerned that would be confusing. We could link to the repo for now, or wait until a page is up.

I'm open to any input.

**Review by Qard — APPROVED** [review]

**Pull request merged**